cmd/csi-resizer/main.go

```diff
@@ -44,8 +45,9 @@ var (
 	resyncPeriod = flag.Duration("resync-period", time.Minute*10, "Resync period for cache")
 	workers      = flag.Int("workers", 10, "Concurrency to process multiple resize requests")
 
-	csiAddress  = flag.String("csi-address", "/run/csi/socket", "Address of the CSI driver socket.")
-	csiTimeout  = flag.Duration("csiTimeout", 15*time.Second, "Timeout for waiting for CSI driver socket.")
+	csiAddress = flag.String("csi-address", "/run/csi/socket", "Address of the CSI driver socket.")
```

Comment:
Since we're going to ignore this csiAddress flag, should we use the [deprecatedflag library](https://github.com/kubernetes-csi/csi-lib-utils/tree/master/deprecatedflags)?

Reply:
wait, why is `csiAddress` being ignored?

Reply:
sorry this got merged before this has chance to get addressed.  but we can do a follow up.